### PR TITLE
[TF-57] Add initial documentation for beta-tensorflow

### PR DESCRIPTION
**Pull request by elezar**
## Description
https://jira.mesosphere.com/browse/TF-57

Still todo:
- [ ] Settle on a version (currently `1.0.0-1.5.0-beta`)
- [ ] Address some TODOs with regards to hyperlinks in the pages

## Urgency
- [ ] Blocker 
- [ ] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

File: pages/services/beta-tensorflow/latest/hdfs/index.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  "service": {
+    "hdfs": {
+      "config_url": "http://mydomain.com/hdfs-config"
+    }
+  }
+}
+```
+This can also be done through the UI.
+
+If you are using a DC/OS {{ hdfs_model.techName }} service with a service name of {{ hdfs_mode.serviceName }}, the endpoint to specify here would be {{ hdfs_mode.endpoint }}.
```
Comment (mpereira) [inline pages/services/beta-tensorflow/latest/hdfs/index.md]: Are these interpolated variables correct? Also notice `hdfs_model` vs `hdfs_mode`.

File: pages/services/beta-tensorflow/latest/install/index.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+}
+```
+
+The service (starting the model training) can then be installed using the command:
+```bash
+$ dcos package install {{ model.packageName }} --options=options.json
+```
+
+<!-- TODO: Check the relative link -->
+For more information on how to define a DC/OS {{ model.techShortName }} model, see the [../model-definition] documentation. More options for configuring the {{ modlel.techName }} service are listed in the [../configuration] documentation.
```
Comment (mpereira) [inline pages/services/beta-tensorflow/latest/install/index.md]: Typo `modlel`.

File: pages/services/beta-tensorflow/latest/release-notes/index.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+- Added support for Marathon-style placement constraints
+
+### Improvements
+<!-- TODO: Is it "airgapped" or "air-gapped" -->
+- Added DC/OS TensorFlow Docker images to resource definition to allow for use in air-gapped clusters
+- Move JDK and Hadoop dependencies out of the Docker
+- Update JRE to 8u162
+- Update bundled Hadoop version to 2.7.5
+- Update default TensorFlow version to 1.5.0
+- Enable caching in the Mesos fetcher
```
Comment (mpereira) [inline pages/services/beta-tensorflow/latest/release-notes/index.md]: `s/Move/Moved/`, `s/Update/Updated/`, etc.

**Comment by benclarkwood** [comment]
If Evan and Murilo are happy, I'm happy.

**Comment by elezar** [comment]
It's definitely not ready yet. The decision was also made to leave this as a community release -- meaning that the docs should probably live elsewhere.

**Comment by imaxxs** [comment]
As discussed we are not releasing the TF docs on dcos docs site hence closing this PR.

**Pull request closed without merging**